graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlServiceBuilder.java

```diff
@@ -252,7 +263,9 @@ public GraphqlService build() {
         for (Consumer<? super DataLoaderRegistry> configurer : dataLoaderRegistries) {
             configurer.accept(dataLoaderRegistry);
         }
-        return new DefaultGraphqlService(builder.build(), dataLoaderRegistry, useBlockingTaskExecutor);
+        errorsHandler = errorsHandler != null ? errorsHandler : GraphqlErrorsHandler.of();
```

Comment:
Let's add `GraphqlErrorsHandler.orElse(another)` method so that we can easily chain the handler:
```java
MyErrorsHandler {

    public HttpResponse handler(...) {
        if (cause instance of MyCause) {
            return myHttpResponse;
        }
        return null; to apply the default handler;
    }
}
```